docs/learning-paths/background-checks/application-requirements.md

```diff
@@ -144,7 +144,7 @@ However, all checks beyond the SSN trace should be conducted in parallel to save
 
 ## What is the step-by-step flow?
 
-Now we can envision the the high-level step-by-step flow that we could expect from the application.
+Now we can envision the the high-level step-by-step flow expected from the application.
```

Comment:
```suggestion
Now we can envision the the high-level step-by-step flow that we expect from the application.
```
We can drop "could", but if an actor can be identified, we strongly prefer active voice over passive voice.